src/applications/medical-copays/containers/DetailPage.jsx

```diff
@@ -1,101 +1,73 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
-import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
 import PDFStatementList from '../components/PDFStatementList';
 import BalanceQuestions from '../components/BalanceQuestions';
 import DisputeCharges from '../components/DisputeCharges';
 import HowToPay from '../components/HowToPay';
 import FinancialHelp from '../components/FinancialHelp';
-import scrollToTop from 'platform/utilities/ui/scrollToTop';
 import { Link } from 'react-router-dom';
 import Modals from '../components/Modals';
 import Alert from '../components/Alerts';
 import { OnThisPage } from '../components/OnThisPage';
 import { formatDate } from '../utils/helpers';
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
 
 const DetailPage = ({ match }) => {
   const selectedId = match.params.id;
-  const error = useSelector(({ mcp }) => mcp.error);
-  const statementData = useSelector(({ mcp }) => mcp.statements);
-  const [selectedCopay] = statementData?.filter(({ id }) => id === selectedId);
-  const [alertType, setAlertType] = useState(null);
+  const statements = useSelector(({ mcp }) => mcp.statements) ?? [];
+  const [selectedCopay] = statements?.filter(({ id }) => id === selectedId);
+  const acctNum = selectedCopay?.pHAccountNumber
+    ? selectedCopay?.pHAccountNumber
+    : selectedCopay?.pHCernerAccountNumber;
 
-  useEffect(
-    () => {
-      scrollToTop();
-      setAlertType(null);
-      if (error) {
-        setAlertType('error');
-      }
-    },
-    [error],
-  );
+  useEffect(() => {
+    scrollToTop();
+  }, []);
 
   return (
     <>
-      <Breadcrumbs className="vads-u-font-family--sans no-wrap">
-        <a href="/">Home</a>
-        <a href="/health-care">Health care</a>
-        <a href="/health-care/pay-copay-bill">Pay your VA copay bill</a>
-        <a href="/health-care/pay-copay-bill/your-current-balances">
-          Your current copay balances
-        </a>
-        <a href="/health-care/pay-copay-bill/your-current-balances/balance-details">
-          Your copay details
-        </a>
-      </Breadcrumbs>
-
       <h1 className="vads-u-margin-bottom--1" data-testid="detail-page-title">
         Your copay bill for {selectedCopay?.station.facilityName}
       </h1>
 
-      {alertType ? (
-        <Alert type={alertType} error={error} />
-      ) : (
-        <>
-          <p className="vads-u-font-size--h3 vads-u-margin-top--0 vads-u-margin-bottom--5">
-            Updated on
-            <span className="vads-u-margin-x--0p5" data-testid="updated-date">
-              {formatDate(selectedCopay?.pSStatementDate)}
-            </span>
-          </p>
+      <p className="vads-u-font-size--h3 vads-u-margin-top--0 vads-u-margin-bottom--5">
+        Updated on
+        <span className="vads-u-margin-x--0p5" data-testid="updated-date">
```

Comment:
Possible to use a `<time datetime=""></time>` element here?

Reply:
yep sure can good catch thanks!